libpacaur/checks.sh

```diff
@@ -393,9 +403,11 @@ ReinstallChecks() {
     # global aurpkgs aurdepspkgs deps aurconflictingpkgs depsAname depsQver depsAver depsAood depsAmain
     depsAtmp=(${depsAname[@]})
     for i in "${!depsAtmp[@]}"; do
+        # check and skip packages with conflicts
         [[ ! $foreign ]] && [[ ! " ${aurpkgs[@]} " =~ " ${depsAname[$i]} " || " ${aurconflictingpkgs[@]} " =~ " ${depsAname[$i]} " ]] && continue
         [[ -z "${depsQver[$i]}" || "${depsQver[$i]}" = '#' || $(vercmp "${depsAver[$i]}" "${depsQver[$i]}") -gt 0 ]] && continue
         [[ ! $installpkg && ! " ${aurdepspkgs[@]} " =~ " ${depsAname[$i]} " ]] && continue
+        # devel packages are never considered to be reinstalled, use always lastest revision without check
```

Comment:
This is correct, but this is primarily due to the design decision of _not_ checking the version of VCS packages before the main prompt (since this operation is so slow). In other word, with `--devel --needed` we don't know at this point if the VCS package will be rebuilt or not - and not knowing isn't a big deal either.